src/components/ExplorePage/index.js

```diff
@@ -0,0 +1,87 @@
+import { Hidden } from "@material-ui/core";
+import { makeStyles } from "@material-ui/core/styles";
+import dynamic from "next/dynamic";
+import PropTypes from "prop-types";
+import React, { useEffect, useState } from "react";
+import useSWR from "swr";
+
+import fetcher from "@/pesayetu/utils/fetcher";
+
+const Map = dynamic(() => import("@/pesayetu/components/ExploreMap"), {
+  ssr: false,
+});
+
+const useStyles = makeStyles(({ breakpoints }) => ({
+  root: {
+    position: "relative",
+    height: "calc(100vh - 88px)",
+    [breakpoints.up("lg")]: {
+      height: "calc(100vh - 110px)",
+    },
+  },
+}));
+
+function ExplorePage({
+  geometries: geometriesProp,
+  geography: geographyProp,
+  ...props
+}) {
+  const classes = useStyles(props);
+  const [geoCode, setGeoCode] = useState(null);
+  const [geometries, setGeometries] = useState(geometriesProp);
+  const [geography, setGeography] = useState(geographyProp);
+  const [shouldFetch, setShouldFetch] = useState(false);
+
+  const { data } = useSWR(
+    shouldFetch
+      ? `https://ng.hurumap.org/api/v1/all_details/profile/1/geography/${geoCode}/?format=json`
+      : null,
+    fetcher
+  );
+
+  useEffect(() => {
+    if (data) {
+      const g = data?.profile.geography;
+      const geom = {
+        boundary: data?.boundary,
+        children: data?.children, // Dictionary keyed by child type
+        parents: data?.parent_layers ?? [], // Array of parent geographies
+        themes: data?.themes ?? [],
+      };
+      setGeometries(geom);
+      setGeography(g);
+    }
```

Comment:
```suggestion
    if (data) {
      const g = data.profile.geography;
      const geom = {
        boundary: data.boundary,
        children: data.children, // Dictionary keyed by child type
        parents: data.parent_layers ?? [], // Array of parent geographies
        themes: data.themes ?? [],
      };
      setGeometries(geom);
      setGeography(g);
    }
```

We've already checked `if (data)`